docs/libertyDev.md

```diff
@@ -193,3 +193,4 @@ These can also be specified as command line parameters in addition to the ones i
 | dockerfile | File | 3.1-M1 | Location of a Dockerfile to be used by dev mode to build the Docker image for the container that will run your Liberty server.  The directory containing the Dockerfile will also be the context for the `docker build`. The default location is `Dockerfile` in the project root. | No |
 | dockerBuildTimeout | integer | 3.1-M3 | Maximum time to wait (in seconds) for the completion of the Docker operation to build the image. The value must be an integer greater than 0. The default value is `60` seconds. | No |
 | skipDefaultPorts | boolean | 3.1-M3 | If set to `true`, dev mode will not publish the default Docker port mappings of `9080:9080` (HTTP) and `9443:9443` (HTTPS). Use this option if you would like to specify alternative local ports to map to the exposed container ports for HTTP and HTTPS using the `dockerRunOpts` parameter. | No |
+| keepTempDockerfile | boolean | 3.1-M3 | If set to `true`, dev mode will not delete the temporary file which is the Dockerfile used to build the container. This file is handy in case you need to debug the process of building the container. The name of the temporary file can be seen when dev mode displays the `docker build` command.| No |
```

Comment:
- The third field should be since version `3.1`.
- Mention that the default value is `false`

Reply:
Ok, I thought we were going to go through and remove all those Milestone markers but now I see that they are actually correct.